### PR TITLE
fix(kit): segmented subscription to control value changes

### PR DESCRIPTION
**Pull request by Vadim2805**
Closes #

### CODE REVIEW

File: projects/kit/components/segmented/segmented.directive.ts [hunk 1]
```diff
@@ -42,12 +43,13 @@ export class TuiSegmentedDirective implements AfterContentChecked, AfterContentI
...
-                switchMap((controls) => merge(controls.map((c) => c.valueChanges))),
+                switchMap((controls) => merge(controls.map((c) => tuiControlValue(c)))),
+                mergeAll(),
                 filter(() => this.isBrowser),
                 takeUntilDestroyed(this.destroyRef),
             )
-            .subscribe(() => {
-                this.update(this.el.querySelector(':checked'));
+            .subscribe((value) => {
+                this.update(this.el.querySelector(`[value=${value}]`));
```
Comment (waterplea) [inline projects/kit/components/segmented/segmented.directive.ts:52]: Why this change?

File: .eslintrc.json
Comment (waterplea) [inline .eslintrc.json:1]: Is that to normalize MacOS vs Windows?

File: projects/kit/components/segmented/segmented.directive.ts [hunk 2]
```diff
@@ -15,9 +15,10 @@ import {RouterLinkActive} from '@angular/router';
 import {EMPTY_QUERY} from '@taiga-ui/cdk/constants';
 import {tuiQueryListChanges} from '@taiga-ui/cdk/observables';
 import {tuiInjectElement} from '@taiga-ui/cdk/utils/dom';
-import {filter, merge, switchMap} from 'rxjs';
+import {filter, merge, mergeAll, switchMap} from 'rxjs';
 
 import {TuiSegmented} from './segmented.component';
+import {tuiControlValue} from '@taiga-ui/cdk/observables';
```
Comment (waterplea) [inline projects/kit/components/segmented/segmented.directive.ts:21]: I think precommit should have moved it up to other imports

File: projects/kit/components/segmented/segmented.directive.ts (hunk 1 above)
Comment (Vadim2805) [reply projects/kit/components/segmented/segmented.directive.ts:52]: Yeh, you are right, we don't need this change.

File: .eslintrc.json
Comment (Vadim2805) [reply .eslintrc.json:1]: yes

File: projects/kit/components/segmented/segmented.directive.ts (hunk 1 above)
Comment (waterplea) [reply projects/kit/components/segmented/segmented.directive.ts:52]: And then `mergeAll()` is unnecessary too

Comment (Vadim2805) [reply projects/kit/components/segmented/segmented.directive.ts]: we still need mergeAll here, tested locally.

File: projects/kit/components/segmented/segmented.directive.ts (hunk 2 above)
Comment (Vadim2805) [reply projects/kit/components/segmented/segmented.directive.ts]: thanks, corrected.

**Review by waterplea — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -80,7 +80,13 @@
...
         ],
         "rules": {
-            "jest/prefer-importing-jest-globals": "off"
+            "jest/prefer-importing-jest-globals": "off",
+            "prettier/prettier": [
+                "error",
+                {
+                    "endOfLine": "auto"
+                }
+            ]
```
Comment (nsbarsukov) [inline package.json:89]: Should it be reverted ? 🤔

**Review by nsbarsukov — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (splincode) [reply package.json:89]: I added this line to taiga linters, it will be updated later
